lib/concurrent/utility/native_extension_loader.rb

```diff
@@ -14,6 +14,10 @@ module NativeExtensionLoader
       def allow_c_extensions?
         Concurrent.on_cruby?
       end
+   
+      def loaded_c_extensions?
+        NativeExtensionLoader.instance_variable_get(:@c_ext_loaded)
```

Comment:
Is the instance_variable_get necessary here? I noticed a few lines below we were not using it.